charts/temporal/templates/server-job.yaml

```diff
@@ -147,6 +147,11 @@ spec:
             - name: TEMPORAL_ADDRESS
               {{- if and (hasKey $.Values.server "internalFrontend") $.Values.server.internalFrontend.enabled }}
               value: {{ include "temporal.fullname" $ }}-internal-frontend.{{ $.Release.Namespace }}.svc:{{ $.Values.server.internalFrontend.service.port }}
+              {{- else if $.Values.server.frontend.ingress.enabled }}
+              {{- range $.Values.server.frontend.ingress.hosts }}
```

Comment:
You are using `range` here. That's a loop over each frontend ingress host, which will result in `value: ..` bring printed multiple times.